.github/workflows/pr-compile-check.yaml

```diff
@@ -0,0 +1,30 @@
+name: 'Pull requests compile checks'
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  workflow_dispatch:
```

Comment:
are you going to implement this workflow manual dispatch or just remove it?

Reply:
@patrick-stephens

Reply:
It is implemented just with no parameters so you can fire it